src/main/java/com/softserve/actent/model/entity/User.java

```diff
@@ -81,31 +75,36 @@
 
     @NonNull
     @ManyToMany
+    @JsonBackReference
```

Comment:
Try to properly use mapper instead of jackson dependencies.

Reply:
done